prql-compiler/src/semantic/resolver.rs

```diff
@@ -1015,7 +1015,11 @@ pub(super) mod test {
             .kind
             .into_transform_call()
             .unwrap_or_else(|e| panic!("Failed to convert `{}`", Expr::from(e)));
-        let exprs = derive.kind.into_derive()?;
+        let exprs = derive
+            .kind
+            .into_derive()
+            .unwrap_or_else(|e| panic!("Failed to convert `{e:?}`"));
```

Comment:
This is a test, we can just `.unwrap()`

Reply:
Right ... I think the intention is to have a nice error message if the `unwrap` fails.